messaging/src/main/java/org/axonframework/eventhandling/processors/streaming/pooled/PooledStreamingEventProcessorConfiguration.java

```diff
@@ -104,17 +106,33 @@ public class PooledStreamingEventProcessorConfiguration extends EventProcessorCo
             eventMessage).reportIgnored();
 
     /**
-     * Constructs a new {@link PooledStreamingEventProcessorConfiguration} with default values.
+     * Constructs a new {@code PooledStreamingEventProcessorConfiguration} with default values.
+     *
+     * @param configuration The configuration, used to retrieve global default values, like
+     *                      {@link MessageHandlerInterceptor MessageHandlerInterceptors}, from.
+     */
+    @Internal
+    public PooledStreamingEventProcessorConfiguration(@Nonnull Configuration configuration) {
+        super(configuration);
+    }
+
+    /**
+     * Constructs a new {@code PooledStreamingEventProcessorConfiguration}.
+     * <p>
+     * This configuration will not have any of the default {@link MessageHandlerInterceptor MessageHandlerInterceptors}
+     * for events. Please use {@link #PooledStreamingEventProcessorConfiguration(Configuration)} when those are desired.
      */
     public PooledStreamingEventProcessorConfiguration() {
```

Comment:
I think it should be marked as @Internal as well, if as you said - it's useful mostly for the tests.